app/javascript/index.js

```diff
@@ -14,6 +15,12 @@ import { lightTheme } from '/themes'
 import './i18n'
 import './styles'
 
+const tagManagerArgs = {
+  gtmId: process.env.GOOGLE_TAG_MANAGER_ID,
+}
+
+TagManager.initialize(tagManagerArgs)
```

Comment:
`GOOGLE_TAG_MANAGER_ID` is marked as optional, what would happen when it is not set?

Reply:
good point, I'll initialise it only if it's set

(it will send a request and get a 404 if it's not set)